src/App.js

```diff
@@ -24,7 +25,7 @@ function App() {
       <Routes>
         <Route path="/login" element={<Login title="login" />} />
         <Route path="/signup" element={<Signup title="register" />} />
-        <Route path="/" element={<Home />} />
+        <Route path="/" element={<Home title="home" />} />
         <Route path="/explore" element={<Explore />}>
           <Route path="" element={<VideoList title="explore" />} />
```

Comment:
Here we can use `index` attribute. `""` path is not required. You can get more info about `index` from react-router docs.

Reply:
great, thanks